lld/test/COFF/lto-weak-undefined.ll

```diff
@@ -0,0 +1,34 @@
+; REQUIRES: x86
+
```

Comment:
Add a file-level comment about the purpose as this may be less intuitive.

Reply:
Thanks, I added this now.